openmessaging/src/main/java/io/openmessaging/rocketmq/MessagingAccessPointImpl.java

```diff
@@ -54,15 +57,76 @@ public Producer createProducer() {
     }
 
     @Override public Consumer createConsumer() {
-        return null;
+        String consumerId = accessPointProperties.getString("CONSUMER_ID");
```

Comment:
Here,do you want to distinguish pullConsumer from pushConsumer by the "CONSUMER_ID" key related value?And do you want set what's string value by using messagingAccessPoint.createPullConsumer method?

Reply:
This is a temporary treatment plan，when you discuss it, then deal with this problem. @zongtanghu @duhengforever